index.js

```diff
@@ -51,6 +52,9 @@ var Omegle = function () {
 		}
 		request(options, function (error, response, body) {
 			if (!error && response.statusCode == 200) {
+				if(!working_servers.indexOf(url)>0){
```

Comment:
Oh sorry, my mistake. One more thing, don't you think the server's list that we're fetching never change so, how about making `updateServer()` like : `url = 'http://front'+Math.floor((Math.random()*100)%16) + 1+'.omegle.com';`